packages/core/src/errorUtils.ts

```diff
@@ -20,6 +20,14 @@ export const getErrorMessage = (error: any | undefined): string => {
     return message;
 };
 
+/**
+ * Will extract the stack from the error, taking the first key found among:
+ * `stacktrace`, `stack`, `componentStack` (component tree for component errors,
+ * which contains only native components names in production).
+ *
+ * In last resort and if `sourceURL`, `line` and `column` are present, it will
+ * generate a stack from these informations.
```

Comment:
"Information" can be only singular if I'm not wrong.

```suggestion
 * generate a stack from this information.
```

Reply:
good catch!